python/tests/test_nessie_cli_error.py

```diff
@@ -1,18 +1,17 @@
-# -*- coding: utf-8 -*-
-"""Authentication tests for Nessi CLI."""
-#  Copyright (C) 2020 Dremio
+# Copyright (C) 2020 Dremio
 #
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-#  http://www.apache.org/licenses/LICENSE-2.0
+# http://www.apache.org/licenses/LICENSE-2.0
 #
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Authentication tests for Nessi CLI."""
```

Comment:
nit: maybe change to "Error handling tests for Nessie CLI." while we are at it, since the existing comment seems copy-paste from the other file.

Reply:
Changed it to the same.